base/system/userinit/lang/de-DE.rc

```diff
@@ -23,24 +31,24 @@ CAPTION "ReactOS LiveCD"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "IDB_LOGO", IDC_STARTLOGO, "Static", WS_CHILD | WS_VISIBLE | SS_OWNERDRAW, 18, 4, 290, 99
-    DEFPUSHBUTTON "Run ReactOS &Live CD", IDC_RUN, 71, 108, 175, 21
-    PUSHBUTTON "&Install ReactOS", IDC_INSTALL, 71, 136, 175, 21
+    DEFPUSHBUTTON "ReactOS &Live CD schalten", IDC_RUN, 71, 108, 175, 21
+    PUSHBUTTON "ReactOS &installieren", IDC_INSTALL, 71, 136, 175, 21
 
     // LTEXT "", IDC_STATIC, 7, 152, 300, 8
 
     CONTROL "", IDC_STATIC, "STATIC", SS_ETCHEDHORZ, 0, 165, 317, 1
-    PUSHBUTTON "&Previous", IDOK, 205, 172, 50, 14
-    PUSHBUTTON "&Cancel", IDCANCEL, 260, 172, 50, 14
+    PUSHBUTTON "&Zurück", IDOK, 205, 172, 50, 14
+    PUSHBUTTON "&Abbrechen", IDCANCEL, 260, 172, 50, 14
 END
 
 STRINGTABLE
 BEGIN
     IDS_SHELL_FAIL "Userinit konnte die Shell nicht starten!"
     IDS_INSTALLER_FAIL "Userinit failed to start the installer!"
```

Comment:
"Userinit konnte das Setup nicht ausführen!"

Reply:
That differs from the upper sentence. Either "ausführen" in both sentences or "starten"